recipes/libmetatensor-torch/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "0.5.0" %}
+{% set url_base = "https://github.com/lab-cosmo/metatensor/releases/download" %}
+
+package:
+  name: libmetatensor-torch
+  version: {{ version }}
+
+source:
+  url: {{ url_base }}/metatensor-torch-v{{ version }}/metatensor-torch-cxx-{{ version }}.tar.gz
+  sha256: 904cf858d8f98b67b948e8a453d8a6da56111e022050d6c8c3d32a9a2cc83464
+
+build:
+  # conda-forge's libtorch is not yet available on Windows
+  skip: true  # [win]
+  number: 0
+  script: |
+    cmake -G Ninja $CMAKE_ARGS -DBUILD_SHARED_LIBS=ON  .
+    cmake --build . --config Release --target install
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - cmake
+    - ninja
+  host:
+    - libmetatensor >=0.1.0,<0.2
+    # we only need the CPU version of libtorch to build the code, but we can run
+    # with either the CPU or CUDA version
+    - libtorch  =*=cpu*
+  run:
+    - libmetatensor >=0.1.0,<0.2
+    - libtorch
+
+test:
+  requires:
+    - cmake
+    - ninja
+    - {{ compiler('cxx') }}
+    - libtorch  =*=cpu*
+  files:
+    - test-project/
+  commands:
```

Comment:
Please add a simple file existence test for checking that the shared library is part of the package and at the correct path.